src/lib/components/primitives/input/input.svelte

```diff
@@ -10,7 +10,7 @@
   export let labelClasses = '';
   export let labelOrder: 'first' | 'last' = 'first';
   export let onChange: (event: Event) => void;
-  export let placeholder: string | null = null;
+  export let placeholder: string | null = ' ';
```

Comment:
Literal space is needed as a default value in order for the floating label to work when we are not provided a placeholder. This was uncovered with the use of stacked labels.